### ISSUE TITLE
2.0 refuse to start with custom path.conf on windows

### ISSUE THREAD

**Issue by babadofar** [elastic/elasticsearch#14963]
In elasticsearch 2.0, I'm trying to set a custom config directory on windows, but get an error message saying "Unrecognized option".
Using the syntax `--path.conf=C:\windows\system32`
According to the list of changes in comand line flags, the double-dash arguments should come at the end.(1) 
It works for me if I edit the elasticsearch.bat file and move the `!newparams!` part to the end

Changing elasticsearch.bat line 46 from 

````
"%JAVA_HOME%\bin\java" %JAVA_OPTS% %ES_JAVA_OPTS% %ES_PARAMS% !newparams! -cp "%ES_CLASSPATH%" "org.elasticsearch.bootstrap.Elasticsearch" start
`````
to
````
"%JAVA_HOME%\bin\java" %JAVA_OPTS% %ES_JAVA_OPTS% %ES_PARAMS% -cp "%ES_CLASSPATH%" "org.elasticsearch.bootstrap.Elasticsearch" start !newparams!
````

[1]. https://www.elastic.co/guide/en/elasticsearch/reference/current/breaking_20_setting_changes.html#_command_line_flags

**Comment by ywelsch** [comment]
This is indeed a bug (see also #13991). How about using `-Des.path.conf=...` in the meanwhile?

**Comment by babadofar** [comment]
thanks, that actually works! I thought I tried it but probably screwed up something else in the meantime;)

**Comment by clintongormley** [comment]
Closed by #15320
